Engine/util/library_posix.h

```diff
@@ -48,8 +48,12 @@ class PosixLibrary : BaseLibrary
 
   AGS::Common::String BuildPath(char *path, AGS::Common::String libraryName)
   {
-    AGS::Common::String platformLibraryName = path;
-    platformLibraryName.Append("/lib");
+    AGS::Common::String platformLibraryName = "";
+    if (path) {
+      platformLibraryName = path;
```

Comment:
Is that true though? Apart from this file, a large amount of code seems to have braces on the same line. I'm more partial to only having braces on new line for function definitions and same line for everything else.  The code feels a bit spread out otherwise.